client/state/inline-help/reducer.js

```diff
@@ -79,4 +81,22 @@ export const search = createReducer(
 	}
 );
 
-export default combineReducers( { requesting, search } );
+export const contactForm = createReducer(
+	{
+		isShowingQandASuggestions: false,
+	},
+	{
+		[ INLINE_HELP_CONTACT_FORM_RESET ]: state => {
+			return Object.assign( {}, state, {
```

Comment:
We can benefit from both the implicit return of the arrow function, and the spread operator here for brevity:
```js
[ INLINE_HELP_CONTACT_FORM_RESET ]: state => ( {
  ...state, 
  isShowingQandASuggestions: false,
} ),
[ INLINE_HELP_CONTACT_FORM_SHOW_QANDA ]: state => ( { 
  ...
} )